src/robot/utils/table_of_content.py

```diff
@@ -0,0 +1,23 @@
+import re
+
+
+def add_toc(doc, importing):
```

Comment:
`importing` is a Boolean, right? Since we cannot use type hints, could perhaps add a default value here to make it more explicit. Could also/alternatively rename it to something more clear. For example, `include_importing_section` would be very explicit but somewhat long.

Reply:
Yes, it's a used as a boolean. If list is empty init is not populated. Adding default is good option, should I actually use `[]` ?

Reply:
If it's used as a Boolean, possible default should be a Boolean as well. On the calling side I'd also use `bool(...)` but that's not such a big deal.